containers/proxy-systemd-services/uyuni-proxy-systemd-services.spec

```diff
@@ -63,51 +63,51 @@ install -D -m 644 uyuni-proxy-services.config %{buildroot}%{_fillupdir}/sysconfi
 %endif
 
 for service in %{SERVICES}; do
-    install -D -m 644 uyuni-${service}.service %{buildroot}%{_unitdir}/uyuni-${service}.service
-    ln -s /usr/sbin/service %{buildroot}%{_sbindir}/rcuyuni-${service}
+    install -D -m 644 ${service}.service %{buildroot}%{_unitdir}/${service}.service
+    ln -s /usr/sbin/service %{buildroot}%{_sbindir}/rc${service}
 done
 
 %check
 
 %pre
 %if !0%{?rhel}
-for service in %{SERVICES}; do
-    %service_add_pre uyuni-${service}.service
-done
+%service_add_pre %{SERVICES}
```

Comment:
As long as you checked RHEL systems supports the services in one line, I am fine with it.

Reply:
Sorry, my comment above is wrong. I wrote it, then noticed my comment was wrong... but didn't remember to remove it before sending the review.